audbackend/core/backend.py

```diff
@@ -96,6 +96,58 @@ def _create(
         """
         raise NotImplementedError()
 
+    def _date(
+            self,
+            path: str,
+            version: str,
+    ) -> str:  # pragma: no cover
+        r"""Get date of file on backend.
+
+        * Return empty string if date cannot be determined
+        * Format should be '%Y/%m/%d'
```

Comment:
Should we move `utils.date_format(date)` and request date is returned as `datetime` object or is this more ambiguous?

Reply:
I though it's easier this way as otherwise the function has to return `None` if it cannot get the date.

Reply:
So I would rather stick with the current implementation or otherwise return the `datetime.datetime` object also with `Backend.date(). But then we have to support `None` as return value.

Reply:
OK, no let's stick with string then. It feels also better to me that `_date()` has the same return type as `date()`